### PR TITLE
Release 1.10

### PR DESCRIPTION
**Pull request by mboersma**
Feedback requested!

### CODE REVIEW

File: _posts/2015-09-03-deis-1-10-faster-builder-and-client.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+
+Due to critical bugs in recent Docker engines related to parallel layer pulls and v1/v2 registry compatibility, we've been forced to pin Deis clusters to Docker 1.5.0.  We now believe these issues have been resolved and are eager to move to the Docker that ships with the latest CoreOS stable release.
+
+### Docker v2 Registry
+
+The Deis registry must be [upgraded to the new v2 API](https://github.com/deis/deis/issues/3814) to ensure forward-compatibility with Docker.  This has a number of added benefits including performance and trust.  However, this will be a tricky migration given the v1 registry data we currently store, as well as the `deis pull` hooks in our fork of the Docker registry.
+
+### Rigger
+
+[Rigger](https://github.com/deis/rigger) is a new tool designed to make it easier to provisioning and test Deis clusters.  With `rigger`, you can stand up and test a Deis cluster on any provider with just a handful of simple commands.  The Deis core team is already using `rigger` internally, but we are looking for feedback.  Please provide your thoughts and requirements on the [design document](https://github.com/deis/deis/issues/4345).
```
Comment (sgoings) [inline _posts/2015-09-03-deis-1-10-faster-builder-and-client.md]: "to provisioning" -> "to provision"

File: _posts/2015-09-03-deis-1-10-faster-builder-and-client.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+## What is Deis?
+
+Deis is an open source PaaS that makes it easy to deploy and manage applications on your own servers. Deis builds upon [Docker](http://docker.io/) and [CoreOS](https://coreos.com/) to provide a lightweight PaaS with a Heroku-inspired workflow.
+
+## 1.10.0 Summary
+
+### New Features
+
+- The `deis` command-line tool has been rewritten in Go and is much faster
```
Comment (sgoings) [inline _posts/2015-09-03-deis-1-10-faster-builder-and-client.md]: Do you want to reference @Joshua-Anderson here?

**Comment by Joshua-Anderson** [comment]
@mboersm Should the app collaborators can no longer manage other collaborators fix/security bug be mentioned?

**Comment by mboersma** [comment]
@Joshua-Anderson sure, I'll add it. I culled anything that didn't seem "big" enough but I agree that qualifies.

File: _posts/2015-09-03-deis-1-10-faster-builder-and-client.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+- Automated provision and upgrade acceptance tests now live under `tests/bin/accept`
+- New contributors can get [free DigitalOcean credits](http://deis.io/code-for-credit-digitalocean/)!
+
+### Improvements
+
+- Various fixes to AWS, Azure, bare metal, Linode, and OpenStack provisioning
+- Optional swarm scheduler components aren't built by default now; use `make -C swarm/ build`
+- NewRelic contrib docs updated now that container CPU and memory usage are available
+- **deis-logger** logs a unique id for each app name, helping Kubernetes scheduler usage
+- **deis-logger** now allows underscores in in container names
```
Comment (bacongobbler) [inline _posts/2015-09-03-deis-1-10-faster-builder-and-client.md]: in its*?

File: _posts/2015-09-03-deis-1-10-faster-builder-and-client.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+- **deis-logspout** fixes a potential deadlock when attaching to containers
+- Integration tests fail if `deis` prints any warning
+- Users can revoke their own app access, and some permissions checks were fixed
+- `deisctl install` and similar command group k8s, mesos, and swarm components logically
```
Comment (bacongobbler) [inline _posts/2015-09-03-deis-1-10-faster-builder-and-client.md]: similar commands*

File: _posts/2015-09-03-deis-1-10-faster-builder-and-client.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+- `deisctl dock` and `deisctl ssh` show help and handle arguments more consistently
+
+### Under the Hood
+
+- All components and clients are built with Go 1.5, which speeds garbage collection and enables concurrent execution
+- nginx was updated to 1.9.4
```
Comment (bacongobbler) [inline _posts/2015-09-03-deis-1-10-faster-builder-and-client.md]: is* so we're in present tense

File: _posts/2015-09-03-deis-1-10-faster-builder-and-client.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+- deis-store components are updated to [Ceph 0.94.3](http://ceph.com/docs/hammer/release-notes/)
+
+For more details, please see [CHANGELOG.md](https://github.com/deis/deis/blob/master/CHANGELOG.md).
+
+## What's Next
+
+### Docker Engine Upgrade
+
+Due to critical bugs in recent Docker engines related to parallel layer pulls and v1/v2 registry compatibility, we've been forced to pin Deis clusters to Docker 1.5.0.  We now believe these issues have been resolved and are eager to move to the Docker that ships with the latest CoreOS stable release.
```
Comment (bacongobbler) [inline _posts/2015-09-03-deis-1-10-faster-builder-and-client.md]: move to the version*? "move to the Docker" sounds a little weird here

**Comment by bacongobbler** [comment]
lgtm

**Comment by carmstrong** [comment]
Should we move the DO credit announcement from a bullet to the intro paragraph? We could probably use more of an oomph and attraction in the opening anyway. Otherwise LGTM.

**Comment by mboersma** [comment]
> Should we move the DO credit announcement from a bullet to the intro paragraph?

Good idea--I did that.

**Comment by carmstrong** [comment]
> Good idea--I did that.

:+1: Love it

**Pull request merged**